src/apigateway/commands/copyUrl.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Window } from '../../shared/vscode/window'
+import { Env } from '../../shared/vscode/env'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { RestApiNode } from '../explorer/apiNodes'
+import * as picker from '../../shared/ui/picker'
+import * as vscode from 'vscode'
+import { ext } from '../../shared/extensionGlobals'
+import { Stage } from 'aws-sdk/clients/apigateway'
+import { ApiGatewayClient } from '../../shared/clients/apiGatewayClient'
+import { RegionProvider } from '../../shared/regions/regionProvider'
+import { DEFAULT_DNS_SUFFIX } from '../../shared/regions/regionUtilities'
+
+const COPY_URL_DISPLAY_TIMEOUT_MS = 2000
```

Comment:
Is this long enough? would 5000 be better?

Reply:
this mirrors the other 2 clipboard-related actions

Reply:
Lets converge them for a consistent experience.

Reply:
Can we use the same timeout const then in a common place?